permissions/plugin-service-fabric.yml

```
@@ -6,4 +6,4 @@ issues:
 paths:
   - "org/jenkins-ci/plugins/service-fabric"
 developers:
-  - "rapatchi"
+  - "dragav"
```

according to the email thread:
```suggestion
  - "azsf_msft"
```